packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -780,3 +776,8 @@ function blend(background: string, overlay: string, opacity: number, gamma: numb
     values: rgb as any,
   });
 }
+
+const removeOpacity = (color: string) => `rgb(from ${color} r g b / 1)`;
+function blendCssVars(background: string, overlay: string, opacity: number) {
+  return `color-mix(in srgb,${background}, ${removeOpacity(overlay)} calc(${opacity} * 100%))`;
```

Comment:
@KenanYusuf What do you think about this approach?

Reply:
@cherniavskii I left a comment here regarding `color-mix` https://github.com/mui/mui-x/pull/14550#discussion_r1766922051

I think it's a great alternative and does exactly what we need besides the couple of versions of Safari that do not support it.